vector/src/main/java/im/vector/app/features/crypto/verification/VerificationBottomSheetViewModel.kt

```diff
@@ -183,6 +184,28 @@ class VerificationBottomSheetViewModel @AssistedInject constructor(
         }
     }
 
+    private fun fetchOtherUserProfile(otherUserId: String) {
+        session.getUser(otherUserId)?.toMatrixItem()?.let {
+            setState {
+                copy(
+                        otherUserMxItem = it
+                )
+            }
+        }
+        // Always fetch the latest User data
+        viewModelScope.launch {
+            tryOrNull { session.userService().resolveUser(otherUserId) }
+                    ?.toMatrixItem()
+                    ?.let {
+                        setState {
+                            copy(
+                                    otherUserMxItem = it
```

Comment:
second change to set up to date data, if possible (network available + API is returning data)